processes/wakeupfd.go

```diff
@@ -77,7 +77,7 @@ func (this *WakeupFd) ReadWakeup() (ret uint64, errno error) {
 // eventfd() is Linux specific
 // C def: int eventfd(unsigned int initval, int flags);
 func eventfd(initval uint, flags int) (fd int, errno error) {
-	val, _, err := syscall.RawSyscall(syscall.SYS_EVENTFD,
+	val, _, err := syscall.RawSyscall(syscall.SYS_EVENTFD2,
```

Comment:
why is this required?  can you give some more context here?

Reply:
To put it short: `aarch64` doesn't have SYS_EVENTFD, so SYS_EVENTFD2 is used instead. 

If I've understood correctly, the SYS_EVENTFD already already actually calls SYS_EVENTFD2 (at least the SYS_EVENTFD doesn't have the flags variable). 

https://man7.org/linux/man-pages/man2/eventfd.2.html